src/lib/Microsoft.Health.Extensions.Fhir.R4/Telemetry/Exceptions/FhirServiceExceptionProcessor.cs

```diff
@@ -41,8 +41,8 @@ public static (Exception customException, string errorName) CustomizeException(E
 
             switch (exception)
             {
-                case FhirException _:
-                    var status = ((FhirException)exception).StatusCode;
+                case FhirClientException _:
```

Comment:
Suggestion:  I know this is an edit to existing code but I think you can inline have a reference to typed variable in the case statement, something like case FhirClientException fhirEx: and this would allow you to avoid the explicit cast.

Reply:
I removed the explicit casts and used the typed variable instead.